platform/core/src/studies/retrieveStudyMetadata.js

```diff
@@ -11,7 +11,7 @@ const StudyMetaDataPromises = new Map();
  * @param {string} studyInstanceUid The UID of the Study to be retrieved
  * @returns {Promise} that will be resolved with the metadata or rejected with the error
  */
-export function retrieveStudyMetadata(server, studyInstanceUid) {
+export function retrieveStudyMetadata(server, studyInstanceUid, filters) {
```

Comment:
Can we update the JSDoc and destructure here so it's clearer which filters can be passed?